lib/cli.js

```diff
@@ -1,30 +1,25 @@
 #!/usr/bin/env node
 'use strict';
-const fs = require('fs');
-const path = require('path');
-const chalk = require('chalk');
-const updateNotifier = require('update-notifier');
-const yosay = require('yosay');
-const stringLength = require('string-length');
-const rootCheck = require('root-check');
-const meow = require('meow');
-const list = require('cli-list');
-const Tabtab = require('tabtab');
-const pkg = require('../package.json');
-const Router = require('./router');
-
+import fs from 'node:fs';
+import path from 'node:path';
+import chalk from 'chalk';
+import updateNotifier from 'update-notifier';
+import yosay from 'yosay';
+import stringLength from 'string-length';
+import rootCheck from 'root-check';
+import meow from 'meow';
+import list from 'cli-list';
+import Tabtab from 'tabtab';
+import pkg from './utils/project-package.js';
+import Router from './router.js';
+import {bootstrap} from 'global-agent';
+import * as routes from './routes/index.js';
+import {getDirname} from './utils/node-shims.js';
+
+const __dirname = getDirname(import.meta.url);
 const gens = list(process.argv.slice(2));
 
-// Override http networking to go through a proxy ifone is configured
-const MAJOR_NODEJS_VERSION = Number.parseInt(process.version.slice(1).split('.')[0], 10);
-
-if (MAJOR_NODEJS_VERSION >= 10) {
-  // `global-agent` works with Node.js v10 and above.
-  require('global-agent').bootstrap();
-} else {
-  // `global-tunnel-ng` works only with Node.js v10 and below.
-  require('global-tunnel-ng').initialize();
-}
+bootstrap();
```

Comment:
We can eliminate this branching because we're > Node 10.
Simply import and call bootstrap.